autogpt_platform/backend/backend/blocks/stripe_link/spend_request.py

```
@@ -64,7 +67,17 @@ async def link_api_request(
             headers=headers,
             json=body,
         )
-        response.raise_for_status()
+        if response.is_error:
+            # Link explains itself in the body; `raise_for_status()` alone
+            # reports "400 Bad Request" and throws that explanation away.
+            try:
+                detail = response.json().get("error", {}).get("message")
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)